tests/privacy_startup_network.py

```diff
@@ -18,8 +19,11 @@
 ###############################################################
 # privacy_startup_network
 #
-# General test for Privacy to verify TLS connections, and slowly adding participants to the security group and verifying
-# how blocks and transactions are sent/not sent.
+# Script implements Privacy Test Case #1. It pairs up producers with p2p connections with relay nodes
+# and the relay nodes connected to at least 2 or more API nodes.  The producers and relay nodes are
```

Comment:
you use `non-producers`, `api nodes`, `relay nodes`. what is the difference?

Reply:
Let me know if the note I added was helpful with this.